dist/bin/scala

```diff
@@ -82,9 +92,12 @@ while [[ $# -gt 0 ]]; do
       shift ;;
     *)
       if [ $execute_script == false ]; then
-        if [[ "$1" == *.scala ]]; then
+        # is a script if extension .scala or .sc or if has scala hash bang
+        if [[ -e "$1" && ("$1" == *.scala || "$1" == *.sc || -f "$1" && `head -n 1 -- "$1" | grep '#!.*scala'`) ]]; then
           execute_script=true
+          [ -n "$SCALA_OPTS" ] && java_options+=($SCALA_OPTS)
           target_script="$1"
+          if [ ! -f $target_script
```

Comment:
Missed closing `]` here.

Reply:
I believe that’s why the CI test failed. Does the `scala` script work locally?

Reply:
I fixed this and pushed the change 30 min. ago, and will push your requested changes within the hour